onnxmltools/convert/common/utils.py

```diff
@@ -284,3 +284,18 @@ def check_input_and_output_types(operator, good_input_types=None, good_output_ty
                 raise RuntimeError('Operator %s (type: %s) got an output %s with a wrong type %s. Only %s are allowed' \
                                    % (operator.full_name, operator.type, variable.full_name, type(variable.type),
                                       good_output_types))
+
+
+def compare_strict_version(v1, v2):
+    if v1 is None:
```

Comment:
It will easily confuse others when v1 is None, since it is not intuitive to define the behavior to compare a version with None.

Reply:
I documented the function which now says:

The function always returns 1 if v1 is None, it is used to set the targeted version to a development version of ONNX.